lib/jormungandr/src/Cardano/Wallet/Jormungandr/Network.hs

```diff
@@ -4,6 +4,7 @@
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE OverloadedLabels #-}
 {-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
Perhaps consider using `NamedFieldPuns` instead? When reading the code, it makes it slightly more obvious where symbols are coming from (at the expense of slightly more typing.)